src/Integration/Laravel/Queue/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ## 0.2.1
 
-### Changed
+### Fixed
```

Comment:
Why not allowing `Changed` anymore ? Not all changes are bug fixes.

Reply:
I would like to have as few "words" as possible

- `BC-Break` and `Added` works well for Major and Minor.
- 'Fixed` works well to.
- for Patches, do you think `Changes` is not ambiguous? (like "I changed the method signatures")

Should we allow `Chores`, `Perf`, `Refactor`, ... like in https://www.conventionalcommits.org/en/v1.0.0/ ?

Reply:
Let's go with https://keepachangelog.com/en/1.1.0/